### DISCUSSION TITLE
How do I get better results on singing voices?

### DISCUSSION THREAD

**Discussion by dillfrescott** [coqui-ai/TTS#1648] [General Q&A] [answered]
When I use voice convert on singing voices, it comes out very robotic sounding. Is there a way to improve this?

**Comment by erogol** [comment]
I've no experience with singing voice, unfortunately. Hope someone else can help.

**Comment by domcross** [comment, marked as answer]
I think the problem is that the TTS model does not learn the pitch. You need to extend the model to have text/phoneme plus pitch information.

Don’t know what your use case is, bit this does look very interesting to me: https://emvoiceapp.com/
